pkg/cloud/services/network/routetables_test.go

```diff
@@ -873,3 +874,73 @@ func (r routeTableInputMatcher) String() string {
 func matchRouteTableInput(input *ec2.CreateRouteTableInput) gomock.Matcher {
 	return routeTableInputMatcher{routeTableInput: input}
 }
+
+func TestService_getRoutesToPublicSubnet(t *testing.T) {
```

Comment:
TODO check if those tests are added in Local Zones PR: https://github.com/kubernetes-sigs/cluster-api-provider-aws/pull/4882

Reply:
From [AWS Docs](https://docs.aws.amazon.com/wavelength/latest/developerguide/wavelength-quotas.html):
> You cannot assign IPv6 addresses to subnets that are in Wavelength Zones.

IPv6 subnets are not supported in Wavelength zones.